src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -611,13 +611,18 @@ public enum ConfigNodes {
 			"false",
 			"",
 			"# When true Towny will handle respawning, with town or resident spawns."),
-	GTOWN_SETTINGS_RESPAWN_PROTECTION(
-			"global_town_settings.respawn_protection",
+	GTOWN_SETTINGS_RESPAWN_PROTECTION_TIME(
+			"global_town_settings.respawn_protection.time",

```

Comment:
Probably good practice to make a root for this since it's being split up. ```GTOWN_SETTINGS_RESPAWN_PROTECTION_ROOT(	"global_town_settings.respawn_protection", "", ""),```